sdk-python/schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE llm_logs (
+    id SERIAL PRIMARY KEY,
+    unixtime_seconds BIGINT NOT NULL,
```

Comment:
Any reason we're using bigint here and not the timestamp type from postgres?

Reply:
I've usually preferred storing datetime values as unixtime - hence the use of BIGINT - however based off an initial search I think you don't get the benefit of datetime operations

I'm not sure which would be preferable, what do you think?

Reply:
> We'll also not want this created from the python side. We'll be creating this at init with aws (So for now, can you run this sql in neon directly)?

okay, will leave the sql file there so we can version control - we can then copy paste it into neon from here?

Reply:
I don't think we should leave it here, as I've just created a MR that will create the table automatically when we setup the aws 👍

Reply:
I think we should go for datetime here for postgres, it could be nice for sorting etc later on

Reply:
I'm thinking perhaps `timestap` is a better name for it too? IF we change the type

Reply:
You can see the [MR here](https://github.com/Theodo-UK/LLM-ops/pull/2/files#diff-8f6f5f166a6362b4ec3c2438edbfd0828f676a132e5698edd40b7f7809005543R18) btw.